src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int TYPE_OF_OPERATION_INDEX = 0;
+    private static final int COST_INDEX = 1;
```

Comment:
It's an amount, not a cost 
<img width="254" alt="image" src="https://user-images.githubusercontent.com/32339836/219013589-f2eede73-db9f-4efc-ae2f-0cfb17b8b061.png">
```suggestion
    private static final int AMOUNT_INDEX = 1;
```